tests/integration/targets/etcd3/defaults/main.yml

```diff
@@ -1,13 +1,7 @@
 ---
```

Comment:
If the file is empty (except comments), please remove it.

Reply:
Interesting... when I'm writing roles, I _never_ remove defaults files. I consider it is in best practises to keep these files even if they are empty. Why else `ansible-galaxy init`should create all of them ?

I'm curious to read your arguments to remove them ?

Reply:
Unnecesary files make tests (and roles) look more complicated than they are.